INSTALL.md

```diff
@@ -7,37 +7,43 @@ BAT on your computer.
 Platforms
 ----------
 
-BAT has been developed on Linux machines running different
-distributions and different versions of the kernel and gcc. As far as
-we know there is nothing distribution-dependent inside of BAT. A gcc
-version >4.2 should suffice to compile the C++ code.
-
-The installation and functionality of BAT has also been tested on Mac
-OS X with gcc and clang.
+BAT has been developed on Linux. The installation, unit tests, and
+examples are run and known to work on Linux and Mac OS X with both gcc
+and clang.
 
```

Comment:
To be fair, we run the tests with GCC on Linux and Clang on OS-X, not with both compilers on both platforms.